plutus-use-cases/test/Spec/Auction.hs

```diff
@@ -271,6 +275,23 @@ finishAuction = do
 prop_FinishAuction :: Property
 prop_FinishAuction = forAllDL finishAuction prop_Auction
 
+-- | This does not hold! The Payout transition is triggered by the sellers off-chain code, so if the
+--   seller walks away the buyer will not get their token (unless going around the off-chain code
+--   and building a Payout transaction manually).
+noLockProof :: NoLockedFundsProof AuctionModel
+noLockProof = NoLockedFundsProof
+  { nlfpMainStrategy   = strat
+  , nlfpWalletStrategy = const strat }
+  where
+    strat = do
+      p <- viewContractState phase
+      when (p == NotStarted) $ action Init
```

Comment:
can't you just stop entirely at this point, rather than initializing the contract at this point?

Reply:
Yes, that would also be a valid strategy.